src/renderer/components/+network-ingresses/ingress-class-set-default.injectable.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import type { IngressClass } from "../../../common/k8s-api/endpoints/ingress-class.api";
+import ingressClassApiInjectable
+  from "../../../common/k8s-api/endpoints/ingress-class.api.injectable";
+import ingressClassStoreInjectable from "./ingress-class-store.injectable";
+
+export const ingressClassSetDefaultInjectable = getInjectable({
+  id: "ingressClassSetDefaultInjectable",
+
+  instantiate(di) {
+    return async (currentItem: IngressClass) => {
+      const api = di.inject(ingressClassApiInjectable);
```

Comment:
nit: dependencies should be `injected` outside of the returned implementation.

That way the timing of being ready is guaranteed.

Reply:
Fixed